sssom/writers.py

```diff
@@ -347,11 +339,11 @@ def get_writer_function(*, output_format: Optional[str] = None, output: str):
 
 
 def write_tables(sssom_dict, output_dir) -> None:
-    for split_id in sssom_dict:
-        sssom_file = os.path.join(output_dir, f"{split_id}.sssom.tsv")
-        msdf = sssom_dict[split_id]
```

Comment:
any time you do this in a loop, you can use the `.items()` iterator over the dict instead